src/styles/layout/_component-grid.scss

```diff
@@ -57,22 +57,13 @@ $breakpoint-accordion-medium-550: 34rem;
     @media (min-width: $breakpoint-scale-up) {
       display: grid;
       grid-template-rows: repeat(4, minmax(0, 0.8fr));
-      grid-template-areas:
-        "accordion dialog"
-        "accordion disclosure"
-        " tabs tabs"
-        "max-scott max-scott";
     }
 
     @media (min-width: $breakpoint-grid-large-screen-1115) {
       grid-template-columns: unset;
       grid-template-columns: 1fr 1fr 1fr;
       grid-template-rows: unset;
       grid-template-rows: repeat(3, minmax(0, 0.8fr));
-      grid-template-areas:
```

Comment:
It's not something you introduced, but I just noticed the unnecessary `unset` values for `grid-template-columns` and `grid-template-rows`. Might as well clean that up while we're in here.